tsconfig.json

```
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "esModuleInterop": true,
+    "module": "esnext",
+    "moduleResolution": "node",
+    "strict": true,
+    "target": "es2018"
+  },
+  "include": [
+    "src/**/*"
+  ]
+}
```

I think we can install `@octokit/tsconfig` as dev dependency and then do this. 

```suggestion
{
  "extends": "@octokit/tsconfig",
  "include": ["src/**/*"],
  "compilerOptions": { "esModuleInterop": true }
}
```

Like we do here: https://github.com/octokit/octokit.js/blob/main/tsconfig.json. We haven't normalized all @octokit repos yet, but that's the new convention

---

done!